src/query-builder/insert-query-builder.ts

```diff
@@ -339,6 +339,65 @@ export class InsertQueryBuilder<DB, TB extends keyof DB, O>
     })
   }
 
+  /**
+   * This can be used to add any additional SQL to the front of the query __after__ the `insert` keyword.
+   *
+   * ### Examples
+   *
+   * ```ts
+   * await db.insertInto('person')
+   *   .modifyFront(sql.raw('-- This is a comment\n'))
+   *   .values(values)
+   *   .execute()
```

Comment:
Is there a real-world use case for this?

Reply:
No, made just for consistency, will remove it